canonical-web-and-design/vanilla-framework
# Move site templates to templates folder

## Done

Move site templates to templates folder for consistency with our other flask project.

Fixes #2896 

## QA

- Pull code
- Run `./run`
- Open http://0.0.0.0:8101/ or [demo]
- Site should work as expected
